README.md

```diff
@@ -240,6 +240,16 @@ spec:
       targetValue: 10 # this will be treated as targetAverageValue
 ```
 
+### Metric weighting based on backend
+
+Skipper supports sending traffic to different backend based on annotations present on the
```

Comment:
Maybe link skipper, if it's not done somewhere else, because it should not be implicit.

Reply:
It's already linked in an earlier section of the README.